src/ServerManager.ts

```diff
@@ -0,0 +1,79 @@
+import { Student } from "./Student";
+import { IServer, UpdateStudentDTO } from "./interface";
+export class ServerManager implements IServer {
+  convertJsonToStudents(json: any): Student[] {
```

Comment:
it's bad practice to use type `any` in ts
please read about types `object`, `Object` and `unknown` in typescript